### ISSUE TITLE
Failed to install playlist

### ISSUE THREAD

**Issue by fizrierahaman** [freeview/iptv#38]
I'm unable to reload the playlist. Display message 'Failed to install playlist'  Error 'Connection Failed'

I'm using TVirl on my Mibox S

**Comment by akmalharith** [comment]
It works fine for me. Did you use the correct URL for the playlist?

**Comment by fizrierahaman** [comment]
Yes sir, I'm using the same url since day one and able to reload until recently that i've face the problem. Will try to reinstall tvirl and setup again.

**Comment by akmalharith** [comment]
Try to ask for help from the developer. https://github.com/stari4ek/tvirl
